### PR TITLE
Testing 1-2-3

### PR DESCRIPTION
**Pull request by riveramj**
Breaking up #18 into separate PRs.

This PR is unit tests for existing models.

Adds tests for:
* Address
* Dimension
* Order Item
* Weight

Adds guards for:
* Address
* Weight
* Dimension

### CODE REVIEW

**Comment by riveramj** [comment]
Develop merged in. Ready for review.

**Comment by natecox** [comment]
@riveramj Sorry for the delay on this, I've been dealing with a family illness issue that has eaten basically all of my time. I'll get back to this as soon as possible.

**Comment by riveramj** [comment]
@natecox no worries. Good luck with everything.

File: shipstation/api.py [hunk 1]
```diff
@@ -110,6 +110,11 @@ def __init__(self, units=None, value=None):
         self.units = units
         self.value = value
 
+        if not value or not isinstance(value, (int, float, Decimal)):
```
Comment (natecox) [inline shipstation/api.py:113]: I know this seems nitpicky, but I strongly prefer that all conditional blocks end with whitespace to show relation (or lack thereof).

```python
if some_condition:
    do_something()

if some_other condition:
    do_something_else()
else:
    fallback()

if a_third_condition:
    do_something()
elif alternative_to_third_condition:
    do_something_else()
else:
    fallback()

```

**Review by natecox — CHANGES_REQUESTED** [review]
Looks good overall, requesting a formatting modification before merging.

**Comment by natecox** [comment]
I'm moderately certain this PR is invalidated by the previous ones I just merged in. Feel free to re-open and correct me if I'm wrong.

**Pull request closed without merging**